requirements/tests.in

```diff
@@ -4,4 +4,5 @@ behave
 coverage
 flake8-tuple
 flake8==3.7.9
-pytest-random-order
\ No newline at end of file
+pre-commit
```

Comment:
I added this in the `tests.in` requirements because these requirements will be renamed to `dev.in` with an upcoming ticket.